src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -122,6 +122,12 @@ SchemaShape IEstimator<CalibratorTransformer<TICalibrator>>.GetOutputSchema(Sche
             return new SchemaShape(outColumns.Values);
         }
 
+        /// <summary>
+        /// Fits the scored <see cref="IDataView"/> creating a <see cref="CalibratorTransformer{TICalibrator}"/>.
+        /// </summary>
+        /// <param name="input"></param>
```

Comment:
><param name="input"></param> [](start = 12, length = 28)

<see cref="IDataView"/> with  "Label" and "Features" columns.
From what I see, we actually don't use Score column at all, we just run predictor on top of feature column to produce "Score" column. So i'm not sure how necessary is to have that Score role in roleMappedData.

Reply:
see line 262. We do use the score column. 

---
In reply to: [248115187](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248115187) [](ancestors = 248115187)

Reply:
During transform stage. During Fit we train calibrator on combination of Predictor + FeatureColumn and LabelColumn. 
But it's just an observation. You don't have to do anything about that.

---
In reply to: [248152774](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248152774) [](ancestors = 248152774,248115187)

Reply:
But you can still put something into empty param, tho

---
In reply to: [248410187](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248410187) [](ancestors = 248410187,248152774,248115187)